config_template.cfg

```diff
@@ -1256,7 +1256,7 @@ FFD_DEGREE= (10, 10, 1)
 FFD_CONTINUITY= 2ND_DERIVATIVE
 %
 % Definition of the FFD planes to be frozen in the FFD (x,y,z).
-% Value from 0 FFD degree in that direction. Pick a value larger than degree if you don't want to fix any plane.
+% Value from 0 FFD degree in that direction. Pick a value larger than degree if you dont want to fix any plane.
```

Comment:
The apostrophe was correct.

Reply:
Seems like he has been a little bit overeager :smile:

Reply:
Gotta save space for :house:

Reply:
got me 👮 ...  I just was annoyed that for my IDE (vscode) this tick `'`opens a block which is never closed.. so the coloring broke for the rest of the file